script/fawe.js

```diff
@@ -29,11 +29,11 @@ var projects = {
 		],
 		"config_path": "plugins"
 	},
-	"Spigot 1.16.5 - 1.18.2": {
+	"Spigot 1.16.5": {
 		"download": [
 			{
 				"label": "Download FAWE",
-				"url": "https://github.com/IntellectualSites/download/raw/gh-pages/artifacts/Fawe/FastAsyncWorldEdit-Bukkit-2.2.1-SNAPSHOT-193.jar",
+				"url": "https://ci.athion.net/job/FastAsyncWorldEdit/588/",
```

Comment:
Ah, might as well link directly to https://ci.athion.net/job/FastAsyncWorldEdit/588/artifact/artifacts/FastAsyncWorldEdit-CLI-2.8.2-SNAPSHOT-588.jar aha

Reply:
i'm sure people would appreciate the direct link to the CLI :p